core-analysis/01-filter-sce.R

```diff
@@ -196,6 +196,7 @@ if (opt$filtering_method == "manual") {
                                         mito_percent_cutoff = opt$mito_percent_cutoff,
                                         detected_gene_cutoff = opt$detected_gene_cutoff,
                                         umi_count_cutoff = opt$umi_count_cutoff)
+  metadata(filtered_sce)$scpca_filter_method <- "Minimum_gene_cutoff"
```

Comment:
```suggestion
  metadata(filtered_sce)$scpca_filter_method <- "manual"
```
So in going back over this, what I think we want to do is keep this here, changing it back to just `manual`, but we should remove the line in the `manual_cell_filtering` function that adds the filtering method to the metadata. 
https://github.com/AlexsLemonade/scpca-downstream-analyses/blob/85df80e018e47f04b0158cd0a4425a6dacf36667/utils/filtering-functions.R#L31